### PR TITLE
Gfx OnOpen() wait for dynvc ready

### PR DESCRIPTION
**Pull request by m4ntis**
Pls merge

### CODE REVIEW

**Review by kubistika — APPROVED** [review]
LGTM

File: server/proxy/pf_rdpgfx.c [hunk 1]
```diff
@@ -30,132 +33,148 @@ static UINT proxy_ResetGraphics(RdpgfxClientContext* context,
...
 {
 	WLog_DBG(TAG, "OnOpen");
-	RdpgfxServerContext* server = (RdpgfxServerContext*) context->custom;
+	proxyData* pdata = (proxyData*) context->custom;
+	RdpgfxServerContext* server = (RdpgfxServerContext*) pdata->ps->gfx;
 
 	if (NULL != do_caps_advertise)
 		*do_caps_advertise = FALSE;
 
+	WaitForSingleObject(pdata->ps->dynvcReady, INFINITE);
```
Comment (speidy) [inline server/proxy/pf_rdpgfx.c:179]: please add comment explaining why we do it here

Comment (speidy) [reply server/proxy/pf_rdpgfx.c:179]: and maybe  a log  before`WaitForSingleObject`

File: server/proxy/pf_rdpgfx.c [hunk 2]
```diff
@@ -200,7 +219,7 @@ static UINT pf_rdpgfx_caps_advertise(RdpgfxServerContext* context,
 		const RDPGFX_CAPSET* currentCaps = &capsAdvertise->capsSets[index];
 
 		/* Add cap to supported caps list if supported by FreeRDP */
-		if(currentCaps->version <= RDPGFX_CAPVERSION_103)
+		if (currentCaps->version <= RDPGFX_CAPVERSION_103)
```
Comment (speidy) [inline server/proxy/pf_rdpgfx.c:227]: should add todo here to support max implemented version

**Review by speidy — CHANGES_REQUESTED** [review]
looks good

**Pull request merged**